ledger/src/blockstore_db.rs

```diff
@@ -539,12 +539,15 @@ impl Rocks {
             &self.write_batch_perf_status,
         );
         let result = self.db.write(batch);
-        if is_perf_enabled {
-            report_rocksdb_write_perf(rocksdb_metric_header!(
-                "blockstore_rocksdb_write_perf,op=write_batch",
-                "write_batch",
-                self.column_options
-            ));
+        if let Some(op_start_instant) = is_perf_enabled {
```

Comment:
probably would be better to rename `is_perf_enabled` to `op_start_instant` where it is defined.